packages/utils/src/lib/reports/__snapshots__/log-stdout-summary.integration.test.ts.snap

```diff
@@ -165,3 +165,159 @@ Lighthouse audits
+● Disallow missing React when using JSX                               passed
+● Disallow negating the left operand of relational operators          passed
+● Disallow passing of children as props                               passed
+● Disallow React to be incorrectly marked as unused                   passed
+● Disallow reassigning \`const\` variables                              passed
+● Disallow the use of \`debugger\`                                      passed
+● Disallow the use of undeclared variables unless mentioned in        passed
+  \`/*global */\` comments
+● Disallow undeclared variables in JSX                                passed
+● Disallow unescaped HTML entities from appearing in markup           passed
+● Disallow usage of deprecated methods                                passed
+● Disallow usage of findDOMNode                                       passed
+● Disallow usage of isMounted                                         passed
+● Disallow usage of the return value of ReactDOM.render               passed
+● Disallow usage of unknown DOM property                              passed
+● Disallow use of optional chaining in contexts where the             passed
+  \`undefined\` value is not allowed
+● Disallow using Object.assign with an object literal as the first    passed
+  argument and prefer the use of object spread instead
+● Disallow using string references                                    passed
+● Disallow variables used in JSX to be incorrectly marked as unused   passed
+● Disallow when a DOM element is using both children and              passed
+  dangerouslySetInnerHTML
+● Enforce a maximum number of lines per file                          passed
+● Enforce camelcase naming convention                                 passed
+● Enforce comparing \`typeof\` expressions against valid strings        passed
+● Enforce consistent brace style for all control statements           passed
+● Enforce ES5 or ES6 class for returning value in render function     passed
+● enforces the Rules of Hooks                                         passed
+● Require \`let\` or \`const\` instead of \`var\`                           passed
+● Require calls to \`isNaN()\` when checking for \`NaN\`                  passed
+● Require or disallow "Yoda" conditions                               passed
+● Require using arrow functions for callbacks                         passed
+
+
+Lighthouse audits
+
+● First Contentful Paint                                              1.2 s
+● Largest Contentful Paint                                            1.5 s
+● Speed Index                                                         1.2 s
+● Cumulative Layout Shift                                             0
+● Total Blocking Time                                                 0 ms
+
+Categories
+
+Category|Score|Audits
+Performance|92|8
+Bug prevention|68|16
+Code style|54|13
```

Comment:
The new `cliui` logs the table like this in `raw` mode, but if we run it it looks normal see img attached. I tried already to get it rendered with borders but did not find a straight forward solution :( .

<img width="670" alt="Screenshot 2024-03-09 at 00 57 59" src="https://github.com/code-pushup/cli/assets/10064416/e2df0af6-330a-47f7-893a-fd2ec642c985">

Reply:
@matejchalk WDYT?

Reply:
The actual output looks good, but having a different output in tests is misleading. Does this test have to use raw mode?

Reply:
I can try to adopt this for this single test.

Reply:
Ok, It's not super clean solution but we see the table now again.
What I don't like is the console mocking and the logs array...

Reply:
I think it's fine, it's only in this one test where it's explained by your comment. And seeing what the users sees in the snapshot is a big benefit 👍